start-grafana.sh

```diff
@@ -90,6 +90,15 @@ if [ $? -eq 0 ]; then
     exit 1
 fi
 
+group_args=()
+is_podman="$(docker --help | grep -o podman)"
+if [ ! -z "$is_podman" ]; then
+    for gid in $(id -G); do
+        group_args+=(--group-add "$gid")
+    done
+    group_args+=(--userns=keep-id)
+fi
```

Comment:
start-grafana.sh worked well for me even without this patch. When is this change needed? Perhaps for the "-G" option to work? Or something else?
And again I understand the userns thing, but not the purpose of the groups thing.